### ISSUE TITLE
Cannot Favorite on Event Page

### ISSUE THREAD

**Issue by Mcscott142** [revelrylabs/bn-mobile-react#124]
## Background 
After implementing the new Scroll functionality I'm unable to favorite an event

**Comment by daybreaker** [comment]
Passed a very basic idea to Brittany. Basically, the header image will be static, the scroll view layer will be on top of that, so it can always scroll over the image, and then the "Back" button and "Favorite" buttons will be on a layer on top of the scroll view. 

We'll need to do some work around to figure out the best way to handle hiding or displaying the buttons as the page scrolls, since theyll now be on top of the scrolling content, but at least now we have 100% functionality, and only have a design decision remaining

**Comment by brittanygay** [comment]
After checking out the changes, I think this looks like a good solution @daybreaker! 

1. We do need to figure out the best way to hide the buttons as the page scrolls, since they are now above the ScrollView content. I can read up more on the z-index bug and see if I can find a workaround because that could be an easy solution to the problem.

2. I can also try adding each icon in it's own `View`, so we can eliminate the issue with the container box blocking access to the ScrollView.

3. After that I'll clean everything up so the design looks good again. 

Picking this up again first thing tomorrow to work on the 3 items mentioned above.

**Comment by brittanygay** [comment]
@daybreaker I pushed up my edits to your branch, and I was able to complete #2 and #3 listed above.

I tried tons of different things, including adding `zIndexes` to try to resolve #1, but I don't think this certain instance is possible at this point? No matter what I try, the buttons will always be covered by the top margin or height that we set on the scrolling section. I've also tried adding back the spacer and giving it a `zIndex` to hide behind the buttons, but that didn't work out either because the scrolling section has to sit ontop of the spacer.

Maybe there's a way to hide the buttons using js once the user scrolls past a certain point. That's the only solution I can think of at this point. I can look at this once more again this week with fresh eyes and see if I can find a solution, but I've already spent a lot of time on it today.

<img width="344" alt="screen shot 2018-10-02 at 2 28 08 pm" src="https://user-images.githubusercontent.com/14582709/46372248-38f40680-c650-11e8-9e07-ee4126923498.png">
